Twig/Extension/OAuthExtension.php

```diff
@@ -25,7 +25,7 @@ final class OAuthExtension extends AbstractExtension
     /**
      * @var OAuthHelper
      */
-    protected $helper;
+    private $helper;
```

Comment:
Small but BC break in fact.

Reply:
In this case it isn't, because it's already a final class. So protected doesn't make sense anyway. Or am I missing something?

Reply:
You're right, didn't notice I made this class `final` ;)